tools/onnx_subgraph/extract_onnx.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) 2025 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import onnx
+import re
+import os
+
+
+def split_subgraph_ios(iofile):
+    iolist = re.split('--input-name |;--output-name ', iofile)
+    in_ = iolist[1].split(';')
+    out_ = iolist[2].split(';')
+    del out_[-1]
+    type = iolist[0].split('subgraph')[0]
+    return in_, out_, type
+
+
+def split_onnx_ios(instrfile,
+                   input_path='net/generation_model_simplify.onnx',
```

Comment:
This may confuse readers of this code with default file name.
can we remove this?